include/libdnf/module/module_item_container.hpp

```diff
@@ -75,10 +79,16 @@ class ModuleItemContainer {
     std::vector<std::string> get_default_profiles(std::string module_name, std::string module_stream);
 
 private:
+    friend ModuleItem;
+
     BaseWeakPtr base;
+    Pool * pool;
+    std::map<std::string, int> repositories;
```

Comment:
Please add a description what repositories contain and how it is used.

Reply:
Or add TODO

Reply:
I moved this to Impl and added a comment.